app/gui2/src/components/ExtendedMenu.vue

```diff
@@ -1,46 +1,49 @@
 <script setup lang="ts">
 import SvgIcon from '@/components/SvgIcon.vue'
+import { isMacLike } from '@/composables/events'
 import { ref } from 'vue'
 
 const isDropdownOpen = ref(false)
 
 const props = defineProps<{
   zoomLevel: number
 }>()
-const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: [] }>()
+const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: []; toggleCodeEditor: [] }>()
+
+const toggleCodeEditorShortcut = isMacLike ? 'Cmd + `' : 'Ctrl + `'
 </script>
 
 <template>
   <div class="ExtendedMenu">
-    <div class="moreIcon" @pointerdown="isDropdownOpen = !isDropdownOpen"></div>
+    <div class="moreIcon" @pointerdown="isDropdownOpen = !isDropdownOpen">…</div>
```

Comment:
Since it is not displayed as an ellipsis but rotated by CSS into the intended shape, I think it would be clearer to use CSS to inject the ellipsis character with `:before`.

Reply:
Replaced with SVg